packages/dx-scheduler-core/src/plugins/common/computeds.ts

```diff
@@ -54,12 +55,12 @@ export const timeScale: TimeScaleFn = (
 };
 
 export const availableViewNames: PureComputed<
-  [string[], string], string[]
-> = (viewNames, viewName) => {
-  if (!viewNames) return [viewName];
-  if (viewNames.findIndex(view => viewName === view) === -1) {
+  [CurrentView[], string, string], CurrentView[]
+> = (viewNames, viewName, viewDisplayName) => {
```

Comment:
The first argument is not a `viewName`. We should use another variable name for it